src/Monocle/Search/Query.hs

```diff
@@ -147,6 +147,7 @@ fields =
     ("group", (fieldText, flavoredField, "Group name")),
     ("branch", (fieldRegex, "target_branch", "Change branch name")),
     ("approval", (fieldText, "approval", "Change approval name")),
+    ("ttm", (fieldNumber, "duration", "Change duration (Time To Merge) in second(s)")),
```

Comment:
We might want to add a couple of test in https://github.com/change-metrics/monocle/blob/833d72ec8a4da358581c3624661ab4808392d3a1/src/Tests.hs#L459 to check the resulting elastic query.

Reply:
Done